ui/pages/routes/routes.component.js

```diff
@@ -596,9 +596,13 @@ export default class Routes extends Component {
           'mouse-user-styles': isMouseUser,
         })}
         dir={textDirection}
-        onClick={() => setMouseUserState(true)}
+        onClick={() => {
```

Comment:
Optional: Also would be very helpful for other devs to understand or modify if we can include some comments for additional logic here 🙏🏻